ui/components/Authentication/Registration/RegisterView.tsx

```diff
@@ -296,18 +296,22 @@ export default function RegisterView(): JSX.Element {
                         }
                         label={
                           <Fragment>
+                            <a
+                            className="hover:text-red-500">
                             I agree to monetr's&nbsp;
                             <a
-                              className="hover:text-blue-500 hover:underline"
+                              className="hover:text-blue-500 underline hover:underline text-black-500"
```

Comment:
https://tailwindcss.com/docs/text-color Also there is not a `text-black-500` class for tailwind?